web/blog/vast-v3.0/index.md

```diff
@@ -0,0 +1,349 @@
+    "pprUser": "tikasrv01\\administrator",
+    "ppParams": "C:\\Windows\\Explorer.EXE",
+    "pssdeep": "49152:2nxldYuopV6ZhcUYehydN7A0Fnvf2+ecNyO8w0w8A7/eFwIAD8j3:Gxj/7hUgsww8a0OD8j3",
+    "pSign": "Signed and has certificate info",
+    "gpFileHash": "CFC6A18FC8FE7447ECD491345A32F0F10208F114B70A0E9D1CD72F6070D5B36F",
+    "gpprUser": "tikasrv01\\administrator",
+    "gpParams": "C:\\Windows\\system32\\userinit.exe",
+    "gpssdeep": "384:YtOYTIcNkWE9GHAoGLcVB5QGaRW5SmgydKz3fvnJYunOTBbsMoMH3nxENoWlymW:YLTVNkzGgoG+5BSmUfvJMdsq3xYu",
+    "gpSign": "Signed",
+    "actRem": "Kill, Rename"
+  }
+}
+```
+
+:::note Syslog Header
+Sometimes CEF is prefixed with a syslog header. VAST currently only supports the
+"raw" form without the syslog header. We are working on support for composable
+*generic* formats, e.g., syslog, where the message can basically be any other
+existing format.
+:::
+
+<!-- TODO: Consider making this a builtin rather than a plugin before the
+release; the CEF plugin has no additional dependencies, so there is little
+reason for it not to be. -->
+
+## Tidbits
+
+This VAST release contains a fair amount of other changes and interesting
+improvements. As always, the [changelog][changelog] contains a complete list of
+user-facing changes since the last release.
+
+Here are some entries that we want to highlight:
+
+[changelog]: //vast.io/changelog#v300
+
+### Removing Empty Fields from JSON Output
+
+The `vast export json` command gained new options in addition to the already
+existing `--omit-nulls`: Pass `--omit-empty-records`, `--omit-empty-lists`,
+or `--omit-empty-maps` to cause VAST not to display empty records, lists, or
+maps respectively.
+
+The flag `--omit-empty` empty combines the three new options and `--omit-nulls`,
+essentially causing VAST not to render empty values at all. To set these options
+globally, add the following to your vast.yaml configuration file:
+
+```yaml
+vast:
+  export:
+    json:
+      # Always omit empty records and lists when using the JSON export format,
+      # but keep empty lists and maps.
+      omit-nulls: true
+      omit-empty-records: true
+      omit-empty-maps: false
+      omit-empty-lists: false
+```
+
+<!-- TODO: Make sure this is actually ready before we release. -->
+
+### Faster Shutdown
+
+VAST processes now shut down much more quickly, which especially improves the
+performance of the `vast import` and `vast export` commands for small amounts of
+data ingested or quickly finishing queries.
+
+### Connection Stability
+
+Client processes now attempt to connect to VAST server processes repeatedly
+until the configured connection timeout expires. We found this to generally
+improve reliability of services with multiple VAST clients, for which we often
+encountered problems with VAST clients being unable to connect to a VAST server
+when started immediately after the VAST server—which at that point in time was
+not ready to acccept the connection yet.
```

Comment:
To me, the main benefit is that you can now start the client before the server, which helps in Docker Compose and other automated environments. If you agree, then it'd be great to lead with this.